www/js/metrics.js

```diff
@@ -1282,7 +1282,11 @@ angular.module('emission.main.metrics',['nvd3',
 
   $scope.setCurDayFrom = function(val) {
     if (val) {
-      $scope.selectCtrl.fromDateTimestamp = moment(val).utc();
+      $scope.selectCtrl.fromDateTimestamp = moment(val).startOf('day');
+      if ($scope.selectCtrl.fromDateTimestamp > $scope.selectCtrl.toDateTimestamp) {
+        var copyToDateTimestamp = $scope.selectCtrl.toDateTimestamp.clone();
```

Comment:
> @sebastianbarry this looks good to me, but can you explain [e-mission/e-mission-docs#758 (comment)](https://github.com/e-mission/e-mission-docs/issues/758#issuecomment-1205451387) further?
> 
> Concretely, you said that
> 
> > To attempt to see what the timestamp is being passed during the setMetrics function. I see that this one in particular is the correct MST timestamp, not a UTC timestamp (I translated the timestamp using an online Unix timestamp translator):
> 
> But it looks like the fix was actually
> 
> > This was happening because the date was being switched to UTC, so it would be changing the date you picked (july 22 @ 23:59:59 MST) to UTC time (july 23 @ 5:59:59 UTC). I removed the UTC switch and that alleviated the problem
> 
> So why didn't you see it as UTC the first time?

--------

That's a great question, and one that I have an answer to.

In my first batch of debugging, I had said:
> To attempt to see what the timestamp is being passed during the setMetrics function. I see that this one in particular is the correct MST timestamp, not a UTC timestamp (I translated the timestamp using an online Unix timestamp translator):

What I was doing to confirm this, was sending a request to the server, then checking the webserver.log. In here, I extracted the timestamp for to/from from my request. Once I had the timestamp, I put it into this online [Epoch Converter - Unix Timestamp Conversion](https://www.epochconverter.com/). This app automatically checks your device's local timezone and gives you the datetime in your local timezone, along with the timestamps real (UTC if the timestamp is in UTC) datetime. _**The timestamps at this point actually were in UTC the whole time, as they were being affected by the `.utc()` function**_

Once I removed the `.utc()` function, I used a the Moment JS function `.format()` while debugging within the actual code, to confirm that these were indeed UTC timestamps, and I used the `.format()` function once I removed the `.utc()` funciton to confirm that the timestamps were being generated as MST timestamps now.

### Does the timestamps being MST now, affect the server retrieving the trips?
As far as I can tell, the server is correctly returning the trips within the correct time range, regardless of the timestamps being MST over UTC now.

Reply:
Changed the vars to consts